src/pages/signin/SAMLSignInPage/index.native.tsx

```diff
@@ -58,6 +83,7 @@ function SAMLSignInPage() {
                 <HeaderWithBackButton
                     title=""
                     onBackButtonPress={() => {
+                        setSAMLUrl('');
```

Comment:
Why do we need to do this?

Reply:
🤔 I was think we needed it for resetting the url if the user leaves and then restarts the SSO sign in process, but it might actually be unnecessary - I'll test

Reply:
It's not needed! Removed!